extension/js/common/composer.ts

```diff
@@ -1201,6 +1208,45 @@ export class Composer {
         if (method === 'forward') {
           this.urlParams.subject = 'Fwd: ' + this.urlParams.subject;
           await this.retrieveDecryptAddForwardedMsg(determined.lastMsgId);
+        } else {
+          try {
+            const { raw } = await Google.gmail.msgGet(this.urlParams.acctEmail, this.urlParams.threadMsgId, 'raw');
+            const msgMimeContent = await Mime.process(Buf.fromBase64UrlStr(raw!));
+            if (!msgMimeContent.from || !msgMimeContent.headers.date || !msgMimeContent.blocks || !msgMimeContent.blocks.length) {
+              return;
+            }
+            if (!this.urlParams.draftId) {
+              const blocksToDisplay = msgMimeContent.blocks.filter(b => b.type === 'encryptedMsg' ||
+                b.type === 'plainText' || b.type === 'plainHtml');
+              const contentToDisplay: Array<string> = [];
+              for (const block of blocksToDisplay) {
+                if (block.type === 'encryptedMsg') {
+                  const armored = Pgp.armor.clip(block.content as string);
+                  if (armored) {
+                    const decrypted = await PgpMsg.decrypt({
+                      kisWithPp: await Store.keysGetAllWithPassphrases(this.urlParams.acctEmail),
+                      encryptedData: Buf.fromUtfStr(armored)
+                    });
+                    if (decrypted.success) {
+                      contentToDisplay.push(decrypted.content.toUtfStr());
+                    }
+                  }
+                } else if (block.type === 'plainHtml') {
+                  contentToDisplay.push(Xss.escape(block.content as string));
```

Comment:
Escaping would result in ugly html tags in the resulting quote text, I think. Can do `Xss.htmlSanitizeAndStripAllTags(content, '\n')` so that you actually convert the html to text, with proper (guessed) newlines.

Let me know if I misunderstood

Reply:
Okay, thanks. `Xss.htmlSanitizeAndStripAllTags(content, '\n')` would be better here